packages/nimble-components/src/utilities/unit-format/default-unit-format.ts

```diff
@@ -69,20 +61,19 @@ export class DefaultUnitFormat extends UnitFormat {
                 unit.scaleFactor,
                 unit.scaledUnitFormatFactory({
                     locale,
-                    intlNumberFormatOptions: this.defaultIntlNumberFormatOptions
-                })
-            );
-            this.leadingZeroScaledUnitFormatters.set(
-                unit.scaleFactor,
-                unit.scaledUnitFormatFactory({
-                    locale,
-                    intlNumberFormatOptions:
-                        this.leadingZeroIntlNumberFormatOptions
+                    // Hack to avoid ts error about roundingPriority not being a known option.
+                    // It has been supported by browsers since 8/23, but TypeScript still hasn't
+                    // added it to the type definitions. See https://github.com/microsoft/TypeScript/issues/56269
+                    intlNumberFormatOptions: {
+                        ...this.defaultIntlNumberFormatOptions,
+                        roundingPriority: 'lessPrecision'
+                    } as Intl.NumberFormatOptions
```

Comment:
Do we need this `as Intl.NumberFormatOptions` cast still? It will hide any real issues with the type.

Reply:
Yes we do, but we will catch any issues with everything other than `roundingPriority` because I am now typing `defaultIntlNumberFormatOptions` as `Intl.NumberFormatOptions`.